frontend/src/app/pages/browse/browse.component.less

```diff
@@ -71,6 +72,7 @@ div.browse-container {
                     a {
                         color: whitesmoke;
                         padding: 5px 5px 2px 2px;
+                        word-break: break-all;
```

Comment:
Our worst-case scenario here is the user with 32 capital Ws for a name. Since the author byline is `inline-block` it won't wrap unless it _really_ has to, but if it does, it'll break wherever it can.